Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:2
```

Comment:
dockerfile should be in sample dir , along the python test as it is for testing only, or named as *dockerfile-test-framework* for example

Reply:
Makes sense. For the Docker daemon to have access to the actual package source when building the image, it does need to be in the root of the project. I'll rename as requested.